src/desktop/components/split_test/running_tests.coffee

```diff
@@ -18,13 +20,15 @@
 module.exports = {
   artist_page_variants:
     key: 'artist_page_variants'
-    outcomes:
-      control: 25
-      no_info: 25
-      no_header: 25
-      no_info_header: 25
+    outcomes: [
```

Comment:
I don't think there's any harm in changing this since it's all just randomly selecting from an array, but can revert this.

Reply:
Yeah I think it's fine as well, looked at the implementation for `_.sample` it's essentially picking a random item for arrays
(See: http://underscorejs.org/docs/underscore.html#section-40)